cli/azd/pkg/project/resources.go

```diff
@@ -118,6 +126,12 @@ func (r *ResourceConfig) UnmarshalYAML(value *yaml.Node) error {
 			return err
 		}
 		raw.Props = cap
+	case ResourceTypeDbMySQL:
+		mp := MySQLProps{}
+		if err := unmarshalProps(&mp); err != nil {
+			return err
+		}
+		raw.Props = mp
```

Comment:
Same here: Is it necessary to add other resource type like: ResourceTypeDbRedis, ResourceTypeDbPostgres, ResourceTypeDbMong?